src/prefect/experimental/cli/worker.py

```diff
@@ -0,0 +1,38 @@
+import typer
+import uvicorn
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.experimental.workers.base import BaseWorker
+from prefect.utilities.dispatch import lookup_type
+
+worker_app = PrefectTyper(
+    name="worker", help="Commands for starting and interacting with workers."
+)
+app.add_typer(worker_app)
+
+
+@worker_app.command()
+async def start(
+    worker_name: str = typer.Option(..., "--name"),
+    worker_pool_name: str = typer.Option(...),
```

Comment:
could it be just `--pool`? a little verbose and a lot to type compared to --name and --type

Reply:
Good call! I've made that change